pkg/config/config.go

```diff
@@ -29,6 +29,14 @@ const ColorOff = "off"
 // ColorAuto represents the auto-state for colors
 const ColorAuto = "auto"
 
+// CredentialFieldNames represents all fields tied to credentials for a profile
+var CredentialFieldNames = [4]string{
```

Comment:
Do we want to delete more than just the credential fields? Presumably you could set certain configs per profile.

Reply:
Oh ok, what other fields should we be removing?

Reply:
Or did you mean we should make it possible to specify a list of additional fields to remove via a flag?

Reply:
Well, more so that there could also be non-credential fields that we should remove on logout (like `color`). Is there a way to do this where it removes any field attached to a profile?

Reply:
I could definitely truncate the entire profile, but is that what we want to do?

I feel like it might lead to unwanted behavior if we are just blindly deleting everything from someone's profile, especially their default profile.

Otherwise if there are other fields, like `color`, that should be removed along with the credentials we can keep a whitelist of them.